MSAL/src/native_auth/network/responses/validator/sign_up/MSALNativeAuthSignUpResponseValidator.swift

```diff
@@ -233,7 +200,6 @@ final class MSALNativeAuthSignUpResponseValidator: MSALNativeAuthSignUpResponseV
             MSALLogger.log(level: .error, context: context, format: "verificationRequired is not supported yet")
             return .unexpectedError
         case .unauthorizedClient,
-             .invalidGrant,
              .expiredToken,
              .userAlreadyExists:
             return .error(apiError)
```

Comment:
Given that the API is restructuring the error responses, can you please double check with them if they're going to return invalid response with many different error codes or if they're going to get rid of this logic. If the latter, we should get rid of the `handleInvalidRequestError(apiError)` method

Reply:
I checked and we will never get an invalid request with 50181, 501811 or 50184 error code. So, we can get rid of the `handleInvalidRequestError(apiError)` method

Reply:
Thanks Danilo for checking with them. I've removed the code related.